deltachat-ios/Controller/ChatListController.swift

```diff
@@ -69,8 +71,9 @@ class ChatListController: UITableViewController {
 
     private var editingConstraints: NSLayoutConstraintSet?
 
-    init(dcContext: DcContext, isArchive: Bool) {
+    init(dcContext: DcContext, dcAccounts: DcAccounts, isArchive: Bool) {
```

Comment:
we don't need to pass both dcAccounts and DcContext there, dcAccounts would be sufficient